packages/event/src/enum/pub-sub.enum.ts

```diff
@@ -74,4 +74,5 @@ export enum PubSubActionEnum {
   bulk_enrollment_mission = 'bulk_enrollment_mission',
   delete_consume = 'delete_consume',
   reset_worksheet = 'reset_worksheet',
+  reset_scorm_enrollment = 'reset_scorm_enrollment',
```

Comment:
```suggestion
  reset_scorm = 'reset_scorm',
```
Acho que pode ser só isso, e podemos usar o type `io.skore.events.mission.enrollment`. O que acha?